site-src/concepts/versioning.md

```diff
@@ -2,59 +2,102 @@
 
 ![Release Channel Overlap](/images/release-channel-overlap.svg)
 <!-- Source: https://docs.google.com/presentation/d/1sfZTV-vlisDUIie_iK_B2HqKia_querT6m6T2_vbAk0/edit -->
 
-Release channels are used to indicate feature stability within Gateway API. For
-example, even if a resource has already graduated to beta, we need a way to test
-out new fields and features before committing to their long term stability. With
-upstream Kubernetes APIs, this is accomplished with [feature
-gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/).
-Unfortunately, CRDs do not have a similar concept yet.
-
-Instead of trying to recreate feature gates in this project, we've introduced
-release channels. Starting in v0.5.0, all new fields and features will start in
-the Experimental release channel. From that point, these may graduate to the
-Standard release channel or be dropped from the API entirely. See
-[GEP-922](/geps/gep-922) for a more detailed discussion of this approach to new
-fields and features.
-
-![Lifecycle of New Gateway API Fields](/images/lifecycle-new-fields.svg)
-<!-- Source: https://docs.google.com/presentation/d/1sfZTV-vlisDUIie_iK_B2HqKia_querT6m6T2_vbAk0/edit -->
+We recommend using the "Standard" release channel by default as it will provide
+a stable experience. The "Experimental" feature enables us to iterate on new
+features quickly but may include breaking changes and should not be relied on
+in production environments.
+
+### API Versions
+Upstream Kubernetes APIs have 3 levels of stability, denoted by alpha, beta, and
+GA API versions. In Gateway API, we've narrowed this down to 2 levels of
+stability, expressed by our release channels as described above.
+
+In general, this means that when resources graduate from the Experimental
+channel to the Standard Channel, they'll also be graduating from an Alpha API
+Version (v1alpha2) to a GA API Version (v1).
+
+#### Rationale
+We're phasing out beta for the following reasons:
+
+1. In most cases there are effectively two levels of API stability - installed
+   by default (stable), and alpha/experimental (unstable). It's not obvious
+   what value an intermediate (Beta) state would have for Gateway API.
+2. The further we separate the "stable" and "experimental" APIs, the longer it
+   takes to get meaningful feedback on new features.
+3. Each unique API version we maintain comes with significant additional cost
+   for users, implementers, and maintainers.
+
+#### Beta
+Although some Gateway API resources already received a Beta API version when
+they graduated to the Standard Channel, that will not be the case for any
+additional resources. All future resources that graduate to Standard Channel
+will include a v1 API version as part of that process.
+
+The resources that already have a beta API version (v1beta1) are:
+
+* HTTPRoute
+* Gateway
+* GatewayClass
+* ReferenceGrant
+
+In the upcoming v1.0 release, HTTPRoute, Gateway, and GatewayClass will all
```

Comment:
This is all good, but we do need a note to update post-1.0. Add it to the pile, I guess.

Reply:
Good call, filed https://github.com/kubernetes-sigs/gateway-api/issues/2457.